src/ddmd/objc.d

```diff
@@ -117,126 +117,140 @@ struct ObjcSelector
     }
 }
 
-struct Objc_ClassDeclaration
+interface Objc
```

Comment:
However in my C++ port, I'll probably include `class Objc { };` with all these virtual functions.  So I'm on the sideline here, as I don't want to deviate the C++ interface too much from upstream, but at the same time, this will be a non-issue once switching to D front-end.